tests/lifecycle/tests/lifecycle_persistent_volume_reclaim_policy.go

```diff
@@ -0,0 +1,253 @@
+
+		dep = deployment.RedefineWithPVC(dep, tsparams.TestVolumeName, tsparams.TestPVCName)
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle-persistent-volume-reclaim-policy test")
+		err = globalhelper.LaunchTests(tsparams.TnfPersistentVolumeReclaimPolicy,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tsparams.TnfPersistentVolumeReclaimPolicy, globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 54206
+	It("One pod with a volume that uses a reclaim policy of recycle [negative]", func() {
+
+		persistentVolume := persistentvolume.RedefineWithPVReclaimPolicy(
+			persistentvolume.DefinePersistentVolume(tsparams.TestPVName, tsparams.LifecycleNamespace), corev1.PersistentVolumeReclaimRecycle)
+
+		err := tshelper.CreatePersistentVolume(persistentVolume, tsparams.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		pvc := persistentvolumeclaim.DefinePersistentVolumeClaim(tsparams.TestPVCName, tsparams.LifecycleNamespace)
+
+		err = tshelper.CreateAndWaitUntilPVCIsBound(pvc, tsparams.LifecycleNamespace, tsparams.WaitingTime, persistentVolume.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Define pod")
+		put := pod.RedefineWithPVC(pod.DefinePod(tsparams.TestPodName, tsparams.LifecycleNamespace,
+			globalhelper.Configuration.General.TestImage), tsparams.TestVolumeName, tsparams.TestPVCName)
+
+		put = pod.RedefinePodWithLabel(put, tsparams.TestTargetLabels)
+
+		err = globalhelper.CreateAndWaitUntilPodIsReady(put, tsparams.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle-persistent-volume-reclaim-policy test")
+		err = globalhelper.LaunchTests(tsparams.TnfPersistentVolumeReclaimPolicy,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tsparams.TnfPersistentVolumeReclaimPolicy, globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 54207
+	It("Two deployments, one with reclaim policy of delete, other with recycle [negative]", func() {
+
+		By("Define & create first pv")
+		persistentVolumea := persistentvolume.RedefineWithPVReclaimPolicy(
+			persistentvolume.DefinePersistentVolume(tsparams.TestPVName, tsparams.LifecycleNamespace), corev1.PersistentVolumeReclaimDelete)
+
+		err := tshelper.CreatePersistentVolume(persistentVolumea, tsparams.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Define & create second pv")
+		persistentVolumeb := persistentvolume.RedefineWithPVReclaimPolicy(
+			persistentvolume.DefinePersistentVolume(tsparams.TestPVName, tsparams.LifecycleNamespace), corev1.PersistentVolumeReclaimRecycle)
```

Comment:
Is this really working? This PV has the same name as the first one (TestPVName).